### ISSUE TITLE
How to use MEL script to change train criterion?

### ISSUE THREAD

**Issue by CPostelnicu** [Microsoft/CNTK#685]
I used the MEL script example provided at [https://github.com/Microsoft/CNTK/issues/430](https://github.com/Microsoft/CNTK/issues/430) to make the ResNet18 pretrained model  work with a regression problem. I need to set the train criterion CE from CrossEntropyWithSoftmax to SquareError. 

I get _"EXCEPTION occurred: Invalid property, criterion, is not supported"_. 

What am I doing wrong?

My code: 

model1 = LoadModel("$ModelDir$/ResNet_18.model")
SetDefaultModel(model1)

inDim=512
labelDimNew=1
fcWScale = 1.13
fcBValue = 0

newL = DnnLayer(inDim, labelDimNew, pool5, fcWScale, fcBValue)
labelsNew = Input(labelDimNew, tag="label")

**newCE = SquareError(labelsNew, newL.z)
SetProperty(newCE, "FinalCriterion", "true")**

**SetInput(newCE, 0, labelsNew)
SetInput(newCE, 1, newL.z)**
SetInput(err, 0, labelsNew)
SetInput(err, 1, newL.z)
SetProperty(newL.z, "output", "true")

**DeleteNode(CE)**
DeleteNode(OutputNodes.z) 
DeleteNode(OutputNodes.t)
DeleteNode(OutputNodes.b)
DeleteNode(OutputNodes.W)
DeleteNode(labels)

Rename(newCE, CE)
Rename(labelsNew, labels)
Rename(newL.z, OutputNodes.z)

SaveModel(model1, "$ModelDir$/ModifiedResNet_18.model", format="cntk")
#DumpModel(model1, "$RootDir$/dump_editedModel.txt", includeData **=true)**

**Comment by CPostelnicu** [comment]
I fixed it by replacing: 
**newCE = SquareError(labelsNew, newL.z)
SetProperty(newCE, "FinalCriterion", "true")**

with this: 

**newCE = SquareError(labels, OutputNodes.z, tag = Criteria)**


where labels and OutputNodes.z are nodes that exist in the original model.  

The rest stays the same. 

Adding the solution in case other newbies in CNTK and ML such as myself should need it.

Closing this.
